File: logger.rb
require 'net/http'

LOGGER_ID = "example"

def logger(message, log_type = "info")
  server_url = "https://hlog.deno.dev"
  uri = URI("#{server_url}/#{LOGGER_ID}/#{log_type}")
  
  Net::HTTP.post(uri, message)
end

logger("Let the logging begin!")

File: logger.dart
import 'package:http/http.dart' as http;

const String loggerID = 'example';

enum LogType { ok, info, warn, error }

void logger(String message, {LogType logType = LogType.info}) {
  final serverURL = 'https://hlog.deno.dev';
  final url = '$serverURL/$loggerID/${logType.toString().split('.').last}';

  http.post(Uri.parse(url), body: message);
}

void main() {
  logger('Let the logging begin! - Dart');
}
